targets/nuttx-stm32f4/jerry_main.c

```diff
@@ -447,58 +409,66 @@ int jerry_main (int argc, char *argv[])
     }
   }
 
-  if (files_counter == 0)
-  {
-    printf ("No input files, running a hello world demo:\n");
-    char *source_p = "var a = 3.5; print('Hello world ' + (a + 1.5) + ' times from JerryScript')";
-
-    jerry_run_simple ((jerry_char_t *) source_p, strlen (source_p), flags);
-    return JERRY_STANDALONE_EXIT_CODE_OK;
-  }
-
   jerry_init (flags);
 
-  register_js_function ("assert", assert_handler);
-  register_js_function ("gc", gc_handler);
+  register_js_function ("assert", jerryx_handler_assert);
```

Comment:
why not reuse `jerryx_handler_register_global` (completely, or in a wrapped form as in jerry-main)?